app/views/gsa18f/procurements/form.html.erb

```diff
@@ -88,7 +88,11 @@
       <%= f.label :additional_info, 'Please include any additional information you feel we need to correctly execute the purchase' %>
       <%= f.text_area :additional_info, class: 'form-control' %>
     </div>
-    <%= f.submit class: 'form-button' %>
+    <% unless ENV['DISABLE_SANDBOX_WARNING'] == 'true' %>
+      <%= f.submit value: 'Submit test request for approval', class: 'form-button' %>
+    <% else %>
+      <%= f.submit class: 'form-button' %>
+    <% end %>
```

Comment:
Since we have the same logic in both forms, can we make a shared partial/helper for it?